Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -1758,6 +1758,7 @@ class ScrollView extends React.Component<Props, State> {
             : this.props.removeClippedSubviews
         }
         key={this.state.contentKey} // TODO(macOS GH#774)
+        inverted={this.props.inverted} // TODO(macOS GH#774)
```

Comment:
just curious does this also exist on windows?

Reply:
We have another Meta internal change achieving the same behavior for Windows. Real native list inversion with correct handling of keyboard or mouse wheel scrolling.
Would be great if we can upstream that change as well.